docs/tutorials/projects/make-a-plant-watering-robot.md

```diff
@@ -170,17 +170,25 @@ Now, select **Yes** to enable SPI:
 Finally, select **Finish**.
 Restart your Pi using `sudo reboot` to make these changes take effect.
 
-### Test your soil moisture readings on your Pi
+### Install the Python SDK
```

Comment:
Please move this to be in the section where the user adds control code and runs that code.  It feels out of place here since they don't actually run any code, so let's at this point continue with the component setup and testing